components/org.wso2.carbon.identity.conditional.auth.functions.utils/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/utils/GetMaskedValueFunctionImpl.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.conditional.auth.functions.utils;
+
+import org.graalvm.polyglot.HostAccess;
+import org.wso2.carbon.identity.central.log.mgt.utils.LoggerUtils;
+
+/**
+ * Implementation of  {@link GetMaskedValueFunction}.
+ */
+public class GetMaskedValueFunctionImpl implements GetMaskedValueFunction {
+
+    @Override
+    @HostAccess.Export
+    public String getMaskedValue(String value) {
+
+        return  LoggerUtils.getMaskedContent(value);
```

Comment:
Shall we fix the formatting here. (Additional space)

Reply:
Addressed with https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/167/commits/4b01845017ca7ef6ca737f08bfdbda6d95f466fe